api/api/src/main/scala/uk/ac/wellcome/platform/api/QueryParams.scala

```diff
@@ -29,9 +30,13 @@ object SingleWorkParams extends QueryParamsUtils {
     parameter(
       (
         "include".as[V2WorksIncludes].?,
+        "expandPaths".as[List[String]].?,
         "_index".as[String].?
       )
     ).tmap((SingleWorkParams.apply _).tupled(_))
+
+  implicit val decodePaths: Decoder[List[String]] =
+    decodeCommaSeparated
```

Comment:
Nice, I didn't know this was a thing!

Reply:
It's defined below, not library functionality